src/tools/spring_simulation_tool.rb

```diff
@@ -1,26 +1,51 @@
-require 'src/system_simulation/modelica_simulation.rb'
-require 'src/system_simulation/modellica_export.rb'
-require 'benchmark'
-
+require_relative 'tool.rb'
 class SpringSimulationTool < Tool
```

Comment:
Do a comment why this is useful as a superclass

Reply:
Done in 6ee258a3c58b1fd820677f52efe4907d283251b1